### PR TITLE
Add initial Docker image for running Voting system 

### PR DESCRIPTION
**Pull request by ptare**
Issue #491 and Issue #486

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Fix node version due to high potential for incompatibilities.
+FROM node:11
+
+# Pull down latest version of code from Github.
+RUN git clone https://github.com/UMAprotocol/protocol.git
+WORKDIR protocol
+
+# Install dependencies and compile contracts.
+RUN npm install
+RUN /bin/bash scripts/buildContracts.sh
```
Comment (mrice32) [inline Dockerfile]: Why do you need the `/bin/bash` here?

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Fix node version due to high potential for incompatibilities.
```
Comment (mrice32) [inline Dockerfile]: Should we put a sample command somewhere (or a script) to run `docker build` with this dockerfile? I'm curious if the user is expected to provide some `context` when running as a part of the command and if they're expected to run it from a particular directory.

File: .dockerignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+node_modules
```
Comment (mrice32) [inline .dockerignore]: I thought this was only necessary if you were using/copying in the local context and using it in the `docker build`? It looks to me like the docker container pulls no files from the local context and it pulls everything it needs from `git clone`, but maybe I'm misunderstanding how the Dockerfile works...?

File: Dockerfile (hunk 1 above)
Comment (ptare) [reply Dockerfile]: Done.

File: Dockerfile (hunk 2 above)
Comment (ptare) [reply Dockerfile]: I added a sample build command, we might have to expand it more later.

File: core/scripts/runVoting.sh [hunk 4]
```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+# Must run from the core/ directory.
+
+while sleep 60; do $(npm bin)/truffle exec ./scripts/Voting.js "$@" &> out.log; done
```
Comment (mrice32) [inline core/scripts/runVoting.sh]: Just curious, do you think `out.log` is more visible than CLI output in GCS or other docker contexts?

Comment (ptare) [reply core/scripts/runVoting.sh]: Not sure, happy to not do the redirect if you think that's better.

Comment (mrice32) [reply core/scripts/runVoting.sh]: I think this is fine - just curious about your thought process.

**Pull request merged**